src/core/account.zig

```diff
@@ -134,7 +140,7 @@ pub fn hashAccount(
     std.mem.writeInt(u64, &int_buf, rent_epoch, .little);
     hasher.update(&int_buf);
 
-    hasher.update(data);
+    while (data.next()) |byte| hasher.update((&byte)[0..1]);
```

Comment:
this will be very slow relative to how it was before -- ref to other iter comment

Reply:
Replaced with faster iterator that iterates over frames fc69c06bd38d6f2dd73fcc3cf311e2cb2881d9cb